### PR TITLE
[Feature] UNO 197 - Dropdown should show status

### PR DESCRIPTION
**Pull request by habemuscode**
**Upgrade version of npm package**

**Related to**

https://github.com/oat-sa/tao-core-ui-fe/pull/173/files

**Don't merge before this PR**

https://github.com/oat-sa/tao-core-ui-fe/pull/173

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -62,7 +62,7 @@
     'label' => 'TAO Base',
     'description' => 'TAO meta-extension',
     'license' => 'GPL-2.0',
-    'version' => '45.7.5',
+    'version' => '45.8.0',
```
Comment (zagovorichev) [inline manifest.php:65]: Don't you need to update version in the package.json as well?

**Review by zagovorichev — CHANGES_REQUESTED** [review]

File: manifest.php (hunk 1 above)
Comment (habemuscode) [reply manifest.php:65]: Need this PR merge before :)

https://github.com/oat-sa/tao-core-ui-fe/pull/173

**Review by jsconan — APPROVED** [review]
code review only

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by adriloma — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Pull request merged**